src/main/groovy/com/palantir/gradle/versions/flexversioning/FlexVersion.groovy

```diff
@@ -23,6 +29,7 @@ class FlexVersion {
     boolean dirty = false;
     boolean tag = false;
 
+
     public FlexVersion (String domain, int commitCount, String gitHash, boolean tag, boolean dirty) {
```

Comment:
my only compliant is the fullVersion variable existing, but you already know that :). Other than that +1 lgtm